src/react/components/common/tagInput/tagInput.scss

```diff
@@ -0,0 +1,126 @@
+@import "../../../../index.scss";
+
+$tagInputWidth: 250px;
+
+.condensed-list-header {
```

Comment:
Did these make these in here by mistake?  Let's keep the condesnsed list css with the rest of it.

Reply:
Yep, mistake. Meant to redirect to use the condensed list css, but forgot